### PR TITLE
commit_bugfix_issues_262_263_290_293_310_331_333

### PR DESCRIPTION
**Pull request by urio999**

### CODE REVIEW

File: core/src/main/java/greencity/controller/EcoNewsController.java [hunk 1]
```diff
@@ -221,17 +225,36 @@ public ResponseEntity<PageableAdvancedDto<EcoNewsGenericDto>> findAll(@ApiIgnore
      * @return PageableDto of {@link EcoNewsDto} instances.
      * @author Danylo Hlynskyi.
      */
+//    @ApiOperation(value = "Find all eco news by page.")
+//    @ApiResponses(value = {
+//        @ApiResponse(code = 200, message = HttpStatuses.OK),
+//        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+//        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED)
+//    })
```
Comment (lagysha) [inline core/src/main/java/greencity/controller/EcoNewsController.java]: Delete this comment

**Review by lagysha — APPROVED** [review]

**Review by DenysLiubchenko — APPROVED** [review]

**Review by staaankey — APPROVED** [review]
Great progress

File: core/src/main/java/greencity/controller/EcoNewsController.java (hunk 1 above)
Comment (urio999) [reply core/src/main/java/greencity/controller/EcoNewsController.java]: Done. Commented code got cleaned.

**Review by OleksandrKobetsDev — APPROVED** [review]
ok

**Pull request merged**